addons/l10n_cl/models/account_journal.py

```diff
@@ -14,6 +14,10 @@ def button_create_new_sequences(self):
         self.ensure_one()
         internal_types = ['invoice', 'debit_note', 'credit_note']
         domain = [('country_id.code', '=', 'CL'), ('internal_type', 'in', internal_types), ('active', '=', True)]
+        if self.company_id.partner_id.l10n_cl_sii_taxpayer_type in ['1', '']:
```

Comment:
Definitely it's not the same, but this is because, when the sequences are put in the journal for the very first time during installation, the company has not been set yet. That way we put correct default document values. If this case is not set this way in the company, it includes undesired active documents in the journal and this is VERY annoying to configure.